README.md

```diff
@@ -2,7 +2,7 @@
 
 ## Description
 
-Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet)
+Collection.sh contains shell functions that are ran in [actions-dotnet](https://github.com/variant-inc/actions-dotnet).
```

Comment:
Won't this be added to actions-nodejs?

Reply:
Yes it will, I will update the description.